packages/contracts-bedrock/meta/VERSIONING.md

```diff
@@ -67,11 +67,11 @@ Versioning for monorepo releases works as follows:
 
 ## Optimism Contracts Manager (OPCM) Versioning
 
-The [OPCM](https://github.com/ethereum-optimism/optimism/blob/main/packages/contracts-bedrock/src/L1/OPContractsManager.sol) is the contract that manages the deployment of all contracts on L1. Its version is the same as the [associated monorepo contracts release](./VERSIONING.md#monorepo-contracts-release-versioning).
+The [OPCM](https://github.com/ethereum-optimism/optimism/blob/main/packages/contracts-bedrock/src/L1/OPContractsManager.sol) is the contract that manages the deployment of all contracts on L1.
```

Comment:
My proposal here was to have the OPCM receive the same version as the associated monorepo contracts release, which would simplify relating one and the other.

Reply:
Would we keep the separate `l1ContractsRelease` OPCM getter and enforce that `opcm.version() == opcm.l1ContractsRelease()`, or would we deprecate one?

Reply:
Deprecating `opcm.version` would mess up with the versioning CI, so if we deprecate one it would need to be `l1ContractsRelease`, which I'm mildly in favour of, not knowing where it is used.

It wouldn't be obvious from the ABI that `opcm.version` matches the contracts release, but it wouldn't be difficult to work out for the users from docs, natspec, and the fact that `opcm.version` would match the contracts release version.

Reply:
Fine by me, I agree that having a snowflake contract is not an appropriate tradeoff for having the OPCM version match the contracts-release version, which is a minor UX improvement.